dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/logging/source/package-info.java

```diff
@@ -11,5 +11,8 @@
 /**
  * @version 1.0.16
  */
+@TraceOptions(traceGroup = "logging", messageBundle = "com.ibm.ws.logging.internal.resources.LoggingMessages")
```

Comment:
should just be able to add this info to the Tr.register call in AccessLogSource.java